data/json/recipes/recipe_deconstruction.json

```diff
@@ -3278,6 +3475,19 @@
     "components": [ [ [ "walnut_unshelled", 1 ] ] ],
     "flags": [ "BLIND_EASY" ]
   },
+  {
+    "type": "uncraft",
+    "result": "whisk",
+    "time": "1 m 30 s",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "scrap", 3 ] ] ]
+  },
+  {
+    "result": "wine_glass",
+    "type": "uncraft",
+    "time": "30 s",
+    "components": [ [ [ "glass_shard", 4 ] ] ]
```

Comment:
This is 120ml of glass while the wine glass only has space for 12ml. I'd suggest dropping it to 2 glass shards and increasing glass volume to compensate for 60ml.